bitcoin/examples/create-p2wpkh-address.rs

```diff
@@ -0,0 +1,27 @@
+extern crate bitcoin;
+
+use bitcoin::{Address, CompressedPublicKey, PrivateKey, Network};
+use bitcoin::secp256k1::{rand, Secp256k1};
+
+// Generate a P2WPKH (pay-to-witness-public-key-hash) address and print it
+// along with the associated private key needed to transact.
+
+fn main() {
+    // Create a new secp256k1 elliptic curve.
+    let s = Secp256k1::new();
+
+    // Generate a secp256k1 secret key.
+    let (secret_key, public_key) = s.generate_keypair(&mut rand::thread_rng());
```

Comment:
This is generating a pair of keys not a secret key.

Reply:
yeah this used to be just the key but I updated it without updating comment, thanks.